Assets/MRTK/Core/Providers/InputSimulation/SimulatedMotionController.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Snapshot of simulated motion controller data.
+    /// </summary>
+    public class SimulatedMotionControllerData
+    {
+        [SerializeField]
+        private bool isTracked = false;
+
+        /// <summary>
+        /// Whether the motion controller is currently being tracked
+        /// </summary>
+        public bool IsTracked => isTracked;
+        [SerializeField]
+        private bool isSelecting = false;
```

Comment:
```suggestion
        public bool IsTracked => isTracked;
        
        [SerializeField]
        private bool isSelecting = false;
```

Reply:
Same for isGrabbing/isPressingMenu - the mismatched set of things are grouped together down below.